include/ground_event.h

```diff
@@ -0,0 +1,26 @@
+#ifndef GUARD_GROUND_EVENT_H
+#define GUARD_GROUND_EVENT_H
+
+#include "structs/str_position.h"
+
+static inline void SetUnkInGroundEvent(const CompactPos *posPtr, PixelPos *dst)
```

Comment:
Needed these for other ground files too. Might need to move this to a different header.

Reply:
You did? Could you share which functions needed that inline?

Reply:
I would check the other `_Add` functions but I used it in `GroundObject_Add`